redis.conf

```diff
@@ -409,6 +409,10 @@ set-proc-title yes
 #
 proc-title-template "{title} {listen-addr} {server-mode}"
 
+# Set the local environment which is used for string comparison operations, and 
+# also affect the performance of Lua scripts. 
+locale ""
```

Comment:
i suppose we should document that an empty string means the locale is derived from the environment variables?

Reply:
Indeed, already done.